rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/10_basic.yml

```diff
@@ -49,16 +49,16 @@ setup:
   - do:
       indices.get_data_stream:
         name: "*"
-  - match: { 0.name: simple-data-stream1 }
-  - match: { 0.timestamp_field.name: '@timestamp' }
-  - match: { 0.generation: 1 }
-  - length: { 0.indices: 1 }
-  - match: { 0.indices.0.index_name: '.ds-simple-data-stream1-000001' }
-  - match: { 1.name: simple-data-stream2 }
-  - match: { 1.timestamp_field.name: '@timestamp2' }
-  - match: { 0.generation: 1 }
-  - length: { 1.indices: 1 }
-  - match: { 1.indices.0.index_name: '.ds-simple-data-stream2-000001' }
+  - match: { data_streams.0.name: simple-data-stream1 }
+  - match: { data_streams.0.timestamp_field.name: '@timestamp' }
+  - match: { data_streams.0.generation: 1 }
+  - length: { data_streams.0.indices: 1 }
+  - match: { data_streams.0.indices.0.index_name: '.ds-simple-data-stream1-000001' }
+  - match: { data_streams.1.name: simple-data-stream2 }
+  - match: { data_streams.1.timestamp_field.name: '@timestamp2' }
+  - match: { data_streams.0.generation: 1 }
+  - length: { data_streams.1.indices: 1 }
+  - match: { data_streams.1.indices.0.index_name: '.ds-simple-data-stream2-000001' }
```

Comment:
I think that at least one of these REST tests should be updated with the full list of fields that are returned from the get data stream API.

Reply:
Thanks Dan, pushed a match for `status` and `template` (the lifecycle policy is tested in `TimeseriesDataStreamIT` as it's an xpack setting)